src/charm.py

```diff
@@ -27,27 +27,26 @@
 
 DATABASE_NAME = "discourse"
 DISCOURSE_PATH = "/srv/discourse/app"
-THROTTLE_LEVELS = {
-    "none": {
-        "DISCOURSE_MAX_REQS_PER_IP_MODE": "none",
-        "DISCOURSE_MAX_REQS_RATE_LIMIT_ON_PRIVATE": "false",
-    },
-    "permissive": {
-        "DISCOURSE_MAX_REQS_PER_IP_MODE": "warn+block",
-        "DISCOURSE_MAX_REQS_PER_IP_PER_MINUTE": 1000,
-        "DISCOURSE_MAX_REQS_PER_IP_PER_10_SECONDS": 100,
-        "DISCOURSE_MAX_USER_API_REQS_PER_MINUTE": 400,
-        "DISCOURSE_MAX_ASSET_REQS_PER_IP_PER_10_SECONDS": 400,
-        "DISCOURSE_MAX_REQS_RATE_LIMIT_ON_PRIVATE": "false",
-    },
-    "strict": {
-        "DISCOURSE_MAX_REQS_PER_IP_MODE": "block",
-        "DISCOURSE_MAX_REQS_PER_IP_PER_MINUTE": 200,
-        "DISCOURSE_MAX_REQS_PER_IP_PER_10_SECONDS": 50,
-        "DISCOURSE_MAX_USER_API_REQS_PER_MINUTE": 100,
-        "DISCOURSE_MAX_ASSET_REQS_PER_IP_PER_10_SECONDS": 200,
-        "DISCOURSE_MAX_REQS_RATE_LIMIT_ON_PRIVATE": "false",
-    },
+THROTTLE_LEVELS: Dict = defaultdict(dict)
```

Comment:
I'm curios for the reason for this change?

Reply:
It allows us to remove one `if`